scala/opentransit/src/main/scala/com/azavea/opentransit/indicators/travelshed/JobsTravelshedIndicator.scala

```diff
@@ -300,9 +294,9 @@ object JobsTravelshedIndicator {
             percentageTile.setDouble(col, row, (sum / totalJobs))
 
           } else {
-            basicTile.setDouble(col, row, Double.NaN)
-            absoluteTile.setDouble(col, row, Double.NaN)
-            percentageTile.setDouble(col, row, Double.NaN)
+            basicTile.setDouble(col, row, 0.0)
+            absoluteTile.setDouble(col, row, 0.0)
+            percentageTile.setDouble(col, row, 0.0)
```

Comment:
I want to double-check that these should be zeroes and not NaN; I'm not very familiar with how this algorithm works, but it looks like this is a catch-all `else` clause; if the semantic meaning here is "no data" or "unknown" then NaN would probably be more appropriate.